conference/ui-js/src/services/current/current.service.js

```diff
@@ -1,7 +1,7 @@
 function Current($resource, $q, $rootScope, LocalStorage) {
   'ngInject';
 
-  const users = $resource('/myinfo', {}, {
+  const users = $resource('/my-info', {}, {
```

Comment:
I think we write own implementation for the whole application. Why You made this changes?

Reply:
i saw local version app. this mistake was refresh to first option